<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilVectorDialog</class>
 <widget class="QDialog" name="UtilVectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vector Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="widthLabel">
     <property name="text">
      <string>Width</string>
     </property>
     <property name="buddy">
      <cstring>widthValue</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="widthValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pixels</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65536</number>
     </property>
     <property name="value">
      <number>1024</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="heightLabel">
     <property name="text">
      <string>Height</string>
     </property>
     <property name="buddy">
      <cstring>heightValue</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="heightValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pixels</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65536</number>
     </property>
     <property name="value">
      <number>768</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="tLabel">
     <property name="text">
      <string>Title</string>
     </property>
     <property name="buddy">
      <cstring>tEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="tEdit">
     <property name="maxLength">
      <number>255</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dLabel">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="buddy">
      <cstring>dEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="dEdit">
     <property name="maxLength">
      <number>255</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QRadioButton" name="saveEntireImage">
     <property name="text">
      <string>Save Entire Image</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QRadioButton" name="saveCurrentView">
     <property name="text">
      <string>Save Current View</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UtilVectorDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UtilVectorDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
